bundles/org.aposin.mergeprocessor/src/org/aposin/mergeprocessor/utils/SvnUtil.java

```diff
@@ -69,8 +70,8 @@
 
 	/**
 	 * Builds a working copy of the target branch of the given mergeUnit. This
-	 * working copy consists at least of the given paths and is most likely not a
```

Comment:
Hey @StefanFranzWeiser what is the Maximum line width specified in eclipse?

Reply:
The default settings of Eclipse is 80. 
It may be possible that the class is not formatted correctly and the auto formatter tries to fix it. If so I would do this within a separat PR. So reduce changes to the fixes you did.

Reply:
Sure.